src/analysis/retail/monk/mistweaver/normalizers/CastLinkNormalizer.ts

```diff
@@ -36,7 +37,7 @@ export const SOOM_GOM = 'SoomGOM';
 export const VIVIFY = 'Vivify';
 
 const RAPID_DIFFUSION_BUFFER_MS = 300;
-const DANCING_MIST_BUFFER_MS = 120;
+const DANCING_MIST_BUFFER_MS = 140;
```

Comment:
Might need to be increased as discussed offline but I'll leave that up to you

Reply:
I will do some more testing and make another PR if 250 looks better